src/components/shared/form_elements/TextFormInput.vue

```diff
@@ -48,18 +52,22 @@ interface Props {
 	modelValue: string | number;
 	autocomplete?: string;
 	inputId: string;
-	placeholder: String;
```

Comment:
IIRC, this is set to a TS String object on purpose because we might want to pass in translation items to it:

`:placeholder="$t( 'placeholder_key' )"`

and TS will throw errors when using the primitive type

Reply:
I changed this because some other tool (probably IDE) complained about `String` being used as a prop type. Since none of our CI tools is complaining and I haven't seen TS errors yet - do you want me to change it back?

Reply:
No, if the linter is happy then I'm happy.